releasenotes/notes/remove-pulse-eb43f66499092489.yaml

```
@@ -0,0 +1,27 @@
+---
+prelude: >
```

The prelude section is special as it's not a list it's basically the header for all the release notes. Using the ungrouped `upgrade` is probably appropriate here. It will put it at the top of the upgrades section.

```suggestion
upgrade: >
```